installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,134 @@
+[id="manually-creating-iam"]
+= Manually creating IAM
+include::modules/common-attributes.adoc[]
+:context: manually-creating-iam
+
+toc::[]
+
+[id="manually-create-iam_{context}"]
+== Manually create IAM
+
+The Cloud Credential Operator can be put into manual mode prior to installation
+in environments where the cloud identity and access management (IAM) APIs are
+not reachable, or the administrator prefers not to store an admin level
+credential secret in the cluster `kube-system` namespace.
+
+.Procedure
+
+. Run the {product-title} installer to generate manifests:
++
+----
+$ openshift-install create manifests --dir=mycluster
+----
+
+. Insert a ConfigMap into the manifests directory so that the Cloud Credential
+Operator will be be placed in manual mode:
++
+----
+$ cat <<EOF > mycluster/manifests/cco-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+EOF
+----
+
+. Remove the admin credential secret created using your local cloud credentials.
+This removal prevents your admin credential from ever being stored in the
+cluster:
++
+----
+$ rm mycluster/openshift/99_cloud-creds-secret.yaml
+----
+
+. Obtain the {product-title} release image your `openshift-install` binary is built
+to use:
++
+----
+$ bin/openshift-install version
+release image quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64
+----
+
+. Locate all `CredentialsRequests` in this release image that target the cloud you
+are deploying on:
++
+----
+$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.z.z-x86_64 --to ./release-image
+----
+
+. Locate the `CredentialsRequests` in the extracted file:
++
+----
+$ grep -l "apiVersion: cloudcredential.openshift.io" * | xargs cat
+----
++
+[NOTE]
+====
+In a future {product-title} release, there will be a new `oc adm release`
+command to scan for the `CredentialsRequests` and display them.
```

Comment:
This is available in 4.6 and should be the recommended option there, maybe with a fallback if people are on older releases. The new command is:

oc adm release extract --credentials-requests --cloud=aws|azure|gcp|openstack|ovirt|vsphere

Reply:
Thanks! I'm noting to include this in our follow-up PR for 4.6.